src/backend/src/prisma/schema.prisma

```diff
@@ -555,3 +563,61 @@ model Manufacturer {
   dateDeleted   DateTime?
   materials     Material[]
 }
+
+enum Design_Review_Status {
+  UNCONFIRMED
+  CONFIRMED
+  SCHEDULED
+  DONE
+}
+
+enum Design_Review_Team {
+  ELECTRICAL
+  SOFTWARE
+  MECHANICAL
+}
+
+model Design_Review {
```

Comment:
these should also be associated with a work package, we can do it on a wbs element and reinforce the logic in the backend service methods (makes it more flexible later on), see how we do it on reimbursement requests